src/algorithms/calorimetry/CalorimeterHitReco.cc

```diff
@@ -39,7 +38,7 @@ using namespace dd4hep;
 
 namespace eicrecon {
 
-void CalorimeterHitReco::init() {
+void CalorimeterHitReco::init(std::shared_ptr<Evaluator_service> evaluator) {
```

Comment:
Can we avoid this? We are (were) moving towards a uniform interface for all algorithms: `void init() final` and `void process(const Input&, const Output&) const final`, so we don't need to rewrite all these factories all the time. Is there a way that we can pass this evaluator service through other means?

@sly2j for potential `algorithms::` interface ideas.

Reply:
Can we register JANA service as algorithms service?